org.openshot.OpenShot.json

```diff
@@ -0,0 +1,385 @@
+                "--disable-static",
+                "--enable-shared",
+                "--disable-doc",
+                "--disable-programs",
+                "--enable-libfontconfig",
+                "--enable-libfreetype",
+                "--enable-libopus",
+                "--enable-librsvg",
+                "--enable-libvpx",
+                "--enable-avresample",
+                "--enable-libx264"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ffmpeg.org/releases/ffmpeg-3.4.1.tar.xz",
+                    "sha256": "5a77278a63741efa74e26bf197b9bb09ac6381b9757391b922407210f0f991c0"
+                }
+            ],
+            "cleanup": [
+                "/share/ffmpeg"
+            ]
+        },
+        {
+            "name": "unittest-cpp",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "post-install": [
+                "mv /app/include/UnitTest++ /app/include/unittest++"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/unittest-cpp/unittest-cpp/releases/download/v2.0.0/unittest-cpp-2.0.0.tar.gz",
+                    "sha256": "1d1b118518dc200e6b87bbf3ae7bfd00a0cfc6be708255f98e5e3d627a7c9f98"
+                }
+            ]
+        },
+        {
+            "name": "imagemagick",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-6.9.9-33.tar.xz",
+                    "sha256": "996adc81c91ebb589c0483b88ce4ddd76e91e0acee4858f71c9d024bda717869"
+                }
+            ],
+            "cleanup": [
+                "/share/doc"
+            ]
+        },
+        {
+            "name": "libopenshot",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "build-options": {
+                "env": {
+                    "UNITTEST_DIR": "/app",
+                    "DESTDIR": "/"
+                }
+            },
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DLIBOPENSHOT_AUDIO_BASE_DIR=/app/include/libopenshot-audio",
+                "-DPYTHON_MODULE_PATH=lib/python3/site-packages"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/OpenShot/libopenshot/archive/v0.1.9.tar.gz",
+                    "sha256": "2547969818092d77a1c742e27cf416b34b4d326abf6b089fc10b921f4a4141ec"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed -i 's|${PYTHON_MODULE_PATH}|lib/python3/dist-packages|g' src/bindings/python/CMakeLists.txt"
```

Comment:
You define `PYTHON_MODULE_PATH` above then replace it?

Reply:
The CMakeLists.txt doesn't take this environment variable into account.